library/entropy.c

```diff
@@ -466,6 +466,80 @@ static int entropy_dummy_source( void *data, unsigned char *output,
     return( 0 );
 }
 
+#if !defined(MBEDTLS_TEST_NULL_ENTROPY) && defined(MBEDTLS_ENTROPY_HARDWARE_ALT)
+
+static int entropy_self_test_bias_gather( unsigned char *buf, size_t buf_len )
```

Comment:
Again, rename the function to be consistent with the name above. I suggest `mbedtls_entropy_source_self_test_gather()`